### DISCUSSION TITLE
Problem with {0} = {1}

### DISCUSSION THREAD

**Discussion by StefanRKarlsson** [maxtoroq/DbExtensions#43] [Q&A] [answered]
Can't get this extension to work like this:

`SQL.SELECT("VoteId").WHERE("{0} = {1}", "columnName", "itemId");`

should it work ?

**Comment by maxtoroq** [comment, marked as answer]
Placeholders are for parameter values or sub-queries. You can do this:

``` csharp
WHERE(columnName + " = {0}", "itemId");
```
